openreview/tools.py

```diff
@@ -78,6 +81,11 @@ def build_groups(conference_group_id, default_params=None):
     return sorted(groups.values(), key=lambda x: len(x.id))
 
 def post_group_parents(client, group, overwrite_parents=False):
+    '''
+    Helper function for posting groups created by build_groups function.
+
+    Recommended that this function be deprecated.
```

Comment:
is there a way to indicate deprecated functions in the doc? an annotation? @mohituniyal ?

Reply:
@mspector could you add @deprecated before the function name?